### PR TITLE
[Crane] Update to snapshot 6995125

### PR DESCRIPTION
**Pull request by JolandaVerhoef**

### CODE REVIEW

File: Crane/buildSrc/src/main/java/com/example/crane/buildsrc/Dependencies.kt [hunk 1]
```diff
@@ -26,46 +26,44 @@ object Libs {
...
             const val runtime = "androidx.compose.runtime:runtime:$version"
             const val runtimeLivedata = "androidx.compose.runtime:runtime-livedata:$version"
             const val material = "androidx.compose.material:material:$version"
             const val foundation = "androidx.compose.foundation:foundation:$version"
             const val layout = "androidx.compose.foundation:foundation-layout:$version"
+            const val tooling = "androidx.compose.ui:ui-tooling:${Compose.version}"
             const val animation = "androidx.compose.animation:animation:$version"
-            const val uiTest = "androidx.ui:ui-test:$version"
+            const val test = "androidx.compose.ui:ui-test:$version"
+            const val uiTest = "androidx.compose.ui:ui-test-junit4:$version"
```
Comment (manuelvicnt) [inline Crane/buildSrc/src/main/java/com/example/crane/buildsrc/Dependencies.kt:59]: Do we need both `test` and `uiTest`?

File: Crane/app/src/androidTest/java/androidx/compose/samples/crane/details/DetailsActivityTest.kt [hunk 2]
```diff
@@ -58,12 +58,20 @@ class DetailsActivityTest {
...
     val composeTestRule = AndroidComposeTestRule(
-        ActivityScenarioRule<DetailsActivity>(
+        activityRule = ActivityScenarioRule<DetailsActivity>(
             createDetailsActivityIntent(
                 InstrumentationRegistry.getInstrumentation().targetContext,
                 testExploreModel
             )
-        )
+        ),
+        activityProvider = { rule ->
```
Comment (manuelvicnt) [inline Crane/app/src/androidTest/java/androidx/compose/samples/crane/details/DetailsActivityTest.kt:67]: Can you add a comment about why is this needed? I'd expect this is not necessary to check

Comment (JolandaVerhoef) [reply Crane/app/src/androidTest/java/androidx/compose/samples/crane/details/DetailsActivityTest.kt:67]: This is currently being discussed in https://issuetracker.google.com/issues/174472899

Comment (manuelvicnt) [reply Crane/app/src/androidTest/java/androidx/compose/samples/crane/details/DetailsActivityTest.kt:67]: Can you add the link as a comment there? Some people might be wondering what's going on :)

**Review by manuelvicnt — APPROVED** [review]
